apps/dashboard/app/views/launchers/editable_form_fields/_editable_select.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <%= render(partial: 'launchers/editable_form_fields/edit_fixed_field',  locals: { form: form, attrib: attrib, fixed: fixed }) %>
 
-    <ol class="list-group text-center col-md-4 mb-3">
+    <ol class="list-group list-group-flush text-center col-md-4 mb-3">
```

Comment:
I don't know if I like the `list-group-flush` here - it seems off to me to only have borders on the bottom/top. 

It seems like we should actually just deal with the overflow in the span right?

Reply:
Also these can `flex` to grow (to a certain size, maybe ~33%?) but it does seem like we should actually just deal with `text-overflow` correctly with potential resizing.